packages/application-components/src/components/modal-pages/custom-form-modal-page/custom-form-modal-page.visualroute.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Formik } from 'formik';
+import {
+  FlameIcon,
+  IconButton,
+  SearchIcon,
+  BinLinearIcon,
+  TextField,
+} from '@commercetools-frontend/ui-kit';
+import { CustomFormModalPage } from '@local-build/application-components';
+import { Suite, Spec } from '../../../../../../visual-testing-app/test-utils';
+
+export const routePath = '/custom-form-modal-page';
+
+const ModalPageWithPortalParentSelector = ({ portalId, ...props }) => (
```

Comment:
```suggestion
const ModalPageWithPortalParentSelector = ({ portalId, ...remainingProps }) => (
```

...otherwise it's easily confused with the full props further down in code.